addons/sys_sem70/CfgAcreSounds.hpp

```diff
@@ -0,0 +1,7 @@
+//CfgAcreSounds.hpp
+
+class CfgAcreSounds {
+    class Acre_SEM70Knob {
+        sound = "\idi\acre\addons\sys_sounds\sounds\semknob.b64";
+    };
+};
```

Comment:
This should be merged into `sys_sounds/CfgAcreSounds.hpp` the SEM52SL does it this way best for consistency